resources/stubs/Relation/ManyToMany.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Cycle\Annotated\Annotation\Relation;
+
+use JetBrains\PhpStorm\ExpectedValues;
+
+final class ManyToMany
+{
+    public function __construct(
+        string $target,
+        string $though,
+        bool $cascade = true,
+        bool $nullable = false,
+        string $innerKey = null,
+        string $outerKey = '{parentRole}_{innerKey}',
+        array $where = [],
+        array $orderBy = [],
+        string $thoughInnerKey = '{sourceRole}_{innerKey}',
+        string $thoughOuterKey = '{targetRole}_{outerKey}',
+        array $thoughWhere = [],
+        bool $fkCreate = true,
+        #[ExpectedValues(values: ['NO ACTION', 'CASCADE', 'SET NULL'])]
+        string $fkAction = 'SET NULL',
+        bool $indexCreate = true,
+        #[ExpectedValues(values: ['lazy', 'eager'])]
+        string $load = 'lazy',
+        // Inverse $inverse = null, // can be uncommented for compatibility with php 8.1
```

Comment:
And what is the point of commenting on this? Does it affects IDE autocomplete behavior?

P.S. Doctrine annotations have supported named arguments since version 1.10+ and the JetBrains plugin quite possibly supports this contstructor format.

Reply:
> Doctrine annotations

This line for attributes only.